Sources/Testing/Events/Recorder/Event.HumanReadableOutputRecorder.swift

```diff
@@ -267,7 +277,8 @@ extension Event.HumanReadableOutputRecorder {
         if issue.isKnown {
           testData.knownIssueCount += 1
         } else {
-          testData.issueCount += 1
+          let issueCount = testData.issueCount[issue.severity] ?? 0
```

Comment:
```suggestion
          let issueCount = testData.issueCount[issue.severity, default: 0]
```

Reply:
I tired that initially but it caused an unrelated compiler crash in `main` snapshot toolchains

Reply:
You filed an issue, yes? :)